src/isomorphic/children/__tests__/onlyChild-test.js

```diff
@@ -35,39 +35,43 @@ describe('onlyChild', function() {
 
   it('should fail when passed two children', function() {
     expect(function() {
-      var instance =
+      var instance = (
         <WrapComponent>
           <div />
           <span />
-        </WrapComponent>;
+        </WrapComponent>
+      );
```

Comment:
Our style guide actually says the opposite for these and we (usually) only wrap with parens at return statements. Might be worth forking the rule or seeing if we can get an option added upstream.